### PR TITLE
Monochromacy is now a 2 points negative quirk.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Title. Moved it from  quirks/neutral.dm to quirks/negative.dm and adjusted its value.

## Why It's Good For The Game
Albeit only slightly, the quirk undermines the player, like the bad touch quirk, only ever-present. Pretty much no one takes it except the occasional roleplayer, and even then it's kinda useless for RP. It's originally supposed to be a neutral trait but Xhuis isn't around anymore anyway.
Other somewhat bad quirks such as Foreigner, Vegetarian and Phobia are also neutral despite being maluses. They'll get their own PRs. I have got plenty of qbp. EDIT: Maybe only Foreigner, after the feature freeze is over.

## Changelog
:cl:
balance: Monochromacy is now a 2 points negative quirk.
/:cl:

### CODE REVIEW

**Comment by interestingusernam3** [comment]
>Albeit only slightly, the quirk undermines the player, like the bad touch quirk, only ever-present. As a result, pretty much no one takes it except the occasional roleplayer.

Seems intended. Check #36426.

>Other somewhat bad quirks such as **Foreigner**, Vegetarian and Phobia are also neutral because of it. They'll get their own PRs. I have got plenty of qbp.

Foreigner is supposed to be neutral. Check #50701.

**Comment by Fikou** [comment]
> > Albeit only slightly, the quirk undermines the player, like the bad touch quirk, only ever-present. As a result, pretty much no one takes it except the occasional roleplayer.
> 
> Seems intended. Check #36426.

who cares if its intended, this is a balance change, not a fix. its a way worse (or at least more annoying) trait than stuff that costs even more than it.

**Comment by Ghommie** [comment]
> 
> Seems intended. Check #36426.
> 
> Foreigner is supposed to be neutral. Check #50701.

Yet my point stands: no one takes them. Most people including me take negative quirks such as Heavy Sleeper, Family Heirloom, Light Drinker and Hypersensitive. They are pretty survivable maluses and offer a good incentive to take them, making them fairly decent quirks that players will use and that won't go unnoticed or unfixed for long if they break.

There's also a funny aspect about the Foreigner quirk. It puts a language barrier between characters who have it and those who don't, forcing them to communicate with each other in different ways; mostly paper (because there's no implementation of language datums in paperwork) but also gestures, emotes. And that's something because 99% of the characters with a ckey in your average round will understand galactic common, including ashwalkers and monkeys (though the latters don't speak it).

**Comment by interestingusernam3** [comment]
>who cares if its intended, this is a balance change, not a fix. its a way worse (or at least more annoying) trait than stuff that costs even more than it.

It's a bad balance change

**Comment by Fikou** [comment]
> > who cares if its intended, this is a balance change, not a fix. its a way worse (or at least more annoying) trait than stuff that costs even more than it.
> 
> It's a bad balance change

why

**Comment by RaveRadbury** [comment]
Please don't touch these quirks.

If you need point values to justify taking them they are not for you.

They are for the people who are willing to take them without getting anything in return.

**Comment by RaveRadbury** [comment]
>And that's something because 99% of the characters with a ckey in your average round will understand galactic common, including ashwalkers and monkeys (though the latters don't speak it).

Ashwalkers lost galcom months ago

**Comment by Fikou** [comment]
> Please don't touch these quirks.
> 
> If you need point values to justify taking them they are not for you.
> 
> They are for the people who are willing to take them without getting anything in return.

How do you know what the intention of the monochromacy quirk is? You are not the creator of it, all the reasons the creator gave were "I don't think it detracts enough from gameplay to be worth a point, but I'll have to consider it." With rebalanced quirk points this is obviously worth a point, it's a downside worse than half the other negative quirks. Nothing here is an argument for keeping it a 0 point quirk.

**Comment by Ghommie** [comment]
> 
> 
> > And that's something because 99% of the characters with a ckey in your average round will understand galactic common, including ashwalkers and monkeys (though the latters don't speak it).
> 
> Ashwalkers lost galcom months ago

Oh, you are right. It's actually some code found outside their language holder for some reason.
I'll correct myself then. 98% of the characters understands galcom.

**Comment by RaveRadbury** [comment]
> How do you know what the intention of the monochromacy quirk is? You are not the creator of it, all the reasons the creator gave were "I don't think it detracts enough from gameplay to be worth a point, but I'll have to consider it." With rebalanced quirk points this is obviously worth a point, it's a downside worse than half the other negative quirks. Nothing here is an argument for keeping it a 0 point quirk.

Fair enough for the contents of this PR, but this is stated as an intended first step towards other quirks, one of which I *did* create.

Not a fan of how this balancing is based around what @Ghommie thinks are quirks that aren't used enough without any evidence of their use rate, while in the same breath also listing his powergaming quirk loadout.

This whole thing is veiled power creep.

**Comment by Timberpoes** [comment]
There have been attempts to give the neutral quirks a point value in the past.

As always, trying to do so gets a VERY strong disagree from me on taking neutral quirks and giving them points values. Neutral quirks are quirks we don't care if players take or not. They can take them for fun, for RP, for a challenge. Or they can not take them at all.

Some exist purely as RP fluff. Foreigner is a big example, but Monochromatic is another example - They're purely RP quirks and are not intended to be balanced against other quirks. They provide an advanced challenge or unique experience for players who want a different take without the station having half a dozen foreigners or colourblind players every shift. Call 'em quirks we don't want players to take for any reason other than the player just wants to for the fun or challenge of it.

**Comment by Fikou** [comment]
> Some exist purely as RP fluff. Foreigner is a big example, but Monochromatic is another example - They're purely RP quirks and are not intended to be balanced against other quirks. 

How do you know this? And how does it make giving it points bad?


> Fair enough for the contents of this PR, but this is stated as an intended first step towards other quirks, one of which I _did_ create.

That's fine, but this is discussion of this pr in specific and i feel it should be kept that way, it doesn't contain other quirks

File: code/datums/quirks/negative.dm [hunk 1]
```diff
@@ -803,3 +803,20 @@
 		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "bad_touch", /datum/mood_event/very_bad_touch)
 	else
 		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "bad_touch", /datum/mood_event/bad_touch)
+
+/datum/quirk/monochromatic
```
Comment (Fikou) [inline code/datums/quirks/negative.dm:807]: i dont know how quirk compatibility works, but this will be incompatible with blind right?

Comment (Ghommie) [reply code/datums/quirks/negative.dm:807]: Yea, I'm making a commit right now. Already noticed, but good eye.

**Comment by tralezab** [comment]
Monochromacy seems like a very low value roleplay engine, as it doesn't effect any part of how you play beyond you boldly announcing that you are in fact Monochromatic (And this has zero effect on your gameplay)

I would argue being a drug addict from quirks is better than Monochromacy for roleplay reasons

**Comment by RaveRadbury** [comment]
>(And this has zero effect on your gameplay)

Monochromacy causes really severe color confusion and can have impact on jobs.

**Comment by tralezab** [comment]
Here's a good idea: I'm gonna play monochromatic today to see for myself

**Comment by Mickyan** [comment]
Weird argument to single out neutral quirks as being roleplay oriented as that's what the whole quirk system as a whole is catered towards, the points system solely exists to disallow players from picking quirks that give an advantage (however minor) without having to also choose a downside, or it simply adds some incentive to picking more negative quirks

As such zero points quirks are for quirks that can be considered double edged swords (deviant tastes), have very little/no easily quantifiable downsides (ageusia) or indirectly encourage behaviour we don't particularly want to encourage (foreigner)

Monochromacy is such a deceptively huge disadvantage and 2 points is nothing, I see no situation where someone would even think about picking it for its points value alone

**Comment by Ghommie** [comment]
> 
> 
> There have been attempts to give the neutral quirks a point value in the past.

Any PR link, I did a quick search but nothing relevant showed up.

> As always, trying to do so gets a VERY strong disagree from me on taking neutral quirks and giving them points values. Neutral quirks are quirks we don't care if players take or not. [...]

Any quirk could potentially fit the description you gave to neutral quirks. Does that make neutral quirks? No. Neutral quirks are objectively quirks with 0 `value`. You could give the main comment at the start of the neutral.dm file a read too, but there's no empiric standard to measure the impact of a quirk to the game.
Also the neutrality of a quirk (like many things in the game) is at the discretion of its original creator (within reason, with a maintainer's approval). It's not unanimous and everlasting. Changes are constantly proposed for the better or worse, regardless of original purposes and intentions.

> Some exist purely as RP fluff. Foreigner is a big example, but Monochromatic is another example - They're purely RP quirks and are not intended to be balanced against other quirks. They provide an advanced challenge or unique experience for players who want a different take without the station having half a dozen foreigners or colourblind players every shift. Call 'em quirks we don't want players to take for any reason other than the player just wants to for the fun or challenge of it.

The RP fluff is one of the slipperiest of the slopes when it comes to multiplayer games such as ss13. Not everyone is HRP, and the fact a feature has some mechanical boons doesn't make it worse for RP or even for the challenge, since nothing stops you from not taking good quirks. Minmaxing is also pretty much irrelevant as long as the advantages of the postive quirks generally cost more than the absolute value of the negative ones. I don't see what's the problem here.

**Comment by interestingusernam3** [comment]
> > >who cares if its intended, this is a balance change, not a fix. its a way worse (or at least more annoying) trait than stuff that costs even more than it.

> > It's a bad balance change

> why

It encourages people to annoy themselves by taking monochromacy because it isn't bad for your combat abilities or your mood management.

**Comment by brotherbeyondd** [comment]
> Here's a good idea: I'm gonna play monochromatic today to see for myself

I play as monochromatic on all my characters :)

It makes chemistry plumbing absolute agony, makes revolution head revs tough to differ from regular revs. Course its a bit harder to recognise people since you don't know their hair colour, tho you get used to it after a while.

Its also a bit harder to see different items if they're all on one tile.

**Comment by Dorsisdwarf** [comment]
I used to play monochromacy for the lols but stopped because it seems like a teeny minor thing and is a COLLOSSAL ass-pain in practice. You never realise just how much of this game is colour-recognition until you're staring at a sea of lumpy grey people moving rapidly

**Pull request closed without merging**